djed-sdk/src/web3.js

```diff
@@ -1,34 +1,45 @@
 import Web3 from "web3";
 import djedArtifact from "./artifacts/DjedABI.json";
 import coinArtifact from "./artifacts/CoinABI.json";
+import { convertInt,web3Promise } from "./helpers";
 
-// Modify getWeb3 to accept BLOCKCHAIN_URI as a parameter
-export const getWeb3 = (BLOCKCHAIN_URI) => {
-  return new Promise((resolve, reject) => {
-    try {
-      const web3 = new Web3(BLOCKCHAIN_URI);
-      resolve(web3);
-    } catch (error) {
-      reject(error);
-    }
-  });
-};
+export const DjedInstance = async (BLOCKCHAIN_URI, DJED_ADDRESS) => {
```

Comment:
Since we have multiple djed deployments, we will need to export multiple djed instances, each with its own blockchain_uri and djed_address.

Reply:
The `ethereum.js` file is designed as  functions that accepts parameters from the env file  and deploys based on the provided variables. Similarly, the djedInstance requires the developer to provide the Djed_address along with the blockchain URI.

However, for stablepay, it needs to be pre-configured with the Milkomeda details provided by the stablepay SDK itself.

Don't you think this approach offers greater flexibility?

Reply:
StablePay will have to work with both the Djed Instance on Milkomeda and the Djed Instance on Ethereum Classic, and also others in the future.

I think the Djed SDK should allow a dev to create a new instance by providing the configuration details for that instance, but the Djed SDK should also provide some preconfigured Djed instances, for the deployments that we already have. Now, whether we create and export these instances in this file or in some other file, I don't know...

Reply:
@ceilican ,I will find a simple staright forward solution to handle both the requirements !